Voycar.Api.Web/Features/Reservation/Endpoints/Get/Personal/Endpoint.cs

```diff
@@ -0,0 +1,65 @@
+namespace Voycar.Api.Web.Features.Reservation.Endpoints.Get.Personal;
+
+using Entities;
+using Repository;
+using Users.Repository;
+
+
+public class Endpoint : Endpoint<Request, Results<Ok<Response>, BadRequest<string>>>
+{
+    private readonly IUsers _userRepository;
+    private readonly IReservations _reservationsRepository;
+
+
+    public Endpoint(IUsers userRepository, IReservations reservationsRepository)
+    {
+        this._userRepository = userRepository;
+        this._reservationsRepository = reservationsRepository;
+    }
+
+
+    public override void Configure()
+    {
+        this.Get("reservation/personal");
```

Comment:
```suggestion
        this.Get(nameof(Reservation).ToLowerInvariant() + "/personal");
```
Ähnlich zur anderen Anmerkung.